.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Let's remove this file to keep the repo agnostic of the personal dev tools we use. I know there are many open-source libraries out there that come with some IDE configs, mostly for VS Code, but for this project, we can leave it up to the developer.

Generally, I add my IDE settings (.e.g. `.idea`, `.vscode`) to my global `.gitignore` to get them ignored in all my repos.

Reply:
Oops, this wasn't supposed to be here. I will edit the `.gitignore` to remove this.